### ISSUE TITLE
Separate Image OCR and Box file features

### ISSUE THREAD

**Issue by Shreeshrii** [tesseract4java/tesseract4java#16]
Currently the tabs are -

- Preprocessing
- Box Editor
- Symbol Overview
- Recognition
- Evaluation

It maybe helpful to keep the tabs related to Image OCR together -

- Preprocessing
- Recognition
- Evaluation

and list the other two related to box file editing later.

**Comment by pvorb** [comment]
Yes, I agree that this may be irritating. However this is generally the order in which you look at the tabs when training a language.

What would you think about a Recognition/basic (Preprocessing, Recognition, Evaluation) vs. Training/advanced mode (all tabs) instead?

**Comment by Shreeshrii** [comment]
I think Recognition/basic (Preprocessing, Recognition, Evaluation) vs. Training/advanced mode (all tabs)  is a good idea. 

Since there maybe some users who are only interested in recognition.
